modules/apps/headless-apio/media-object/media-object-apio-client-test/src/testIntegration/java/com/liferay/media/object/apio/client/test/MediaObjectApioTest.java

```diff
@@ -204,6 +213,43 @@ public void testCreateDocumentInFolder() {
 		);
 	}
 
+	@Test
+	public void testCreateImageInDocumentRepository() {
+		_documentHref = ApioClientBuilder.given(
```

Comment:
Are we overring the value _documentHref? in atest

Reply:
Yes!